app/controllers/companies_controller.rb

```diff
@@ -1,53 +1,19 @@
-class CompaniesController < ApplicationController
-  before_action :set_company, only: [:show, :update, :destroy]
+class CompaniesController < CrudController
+  self.permitted_attrs = %i[name web email phone
+                            partnermanager contact_person email_contact_person
+                            phone_contact_person crm level my_company]
+
+  self.nested_models = %i[locations employee_quantities people]
 
-  # GET /companies
   def index
     @companies = Company.all
-
     render json: @companies
```

Comment:
@vars are instance variables. in that case u don't need an instance variable, do you ?
i would just calll render json: Company.all in that case. there's no reason for a variable here :)

update: do we need that method ? isn't this default index action as defined in crud controller ?
i think you just have to add a :list scope to Company model. See Activity Model for example